flask_pydantic_spec/types.py

```diff
@@ -1,5 +1,6 @@
 import re
-from typing import Optional, Type, Iterable, Mapping, Any, Dict
+from collections import namedtuple
+from typing import Optional, Type, Iterable, Mapping, Any, Dict, is_typeddict, NamedTuple
```

Comment:
I think `is_typeddict` is only available for 3.10 on - `typing_extensions` should give access to this though - feel free to add it to the requirements and then import from that instead?

Reply:
Ah. Sure I'll fix that

Reply:
Fixed.